charts/core/README.md

```diff
@@ -86,10 +86,10 @@ Chart container component.
 
 | Name                        | Description                                                                   | Type                                   | Default                          |
 | --------------------------- | ----------------------------------------------------------------------------- | -------------------------------------- | -------------------------------- |
+| `groupId` _(optional)_      | Charts with the same `groupId` will have their tooltips synced across charts. | `string`                               |                                  |
 | `onChartReady`              | Callback to be called when chart is finished rendering.                       | `() => void`                           |                                  |
-| `zoomSelect` _(optional)_   | Enable zoom select (click and drag area selection) for either axis.           | `{ xAxis?: boolean; yAxis?: boolean }` | `{ xAxis: false, yAxis: false }` |
 | `onZoomSelect` _(optional)_ | Callback to be called when a zoom selection is made if enabled.               | `(e: ZoomSelectionEvent) => void`      |                                  |
-| `groupId` _(optional)_      | Charts with the same `groupId` will have their tooltips synced across charts. | `string`                               |                                  |
+| `zoomSelect` _(optional)_   | Enable zoom select (click and drag area selection) for either axis.           | `{ xAxis?: boolean; yAxis?: boolean }` | `{ xAxis: false, yAxis: false }` |
```

Comment:
Should we be more explicit that`onZoomSelect` will do nothing if nothing is passed to this prop?

Reply:
That was my intention when I noted "if enabled" in the `onZoomSelect` description. Does that seem like not enough?

I see this like `Code`'s `copyable` and `onCopy`. Someone could add an `onCopy` handler, and if they never enable `copyable` it won't do anything https://github.com/mongodb/leafygreen-ui/blob/main/packages/code/README.md

Reply:
I think if I didn't have any context, I would not know right away what "if enabled" meant. Maybe something like:

> "Callback to be called when a zoom selection is made if `zoomSelect` is enabled"

Reply:
Updated! That does seem clearer. I think my lack of concern was that like in the `Code` docs, we don't mention anything about it. To me it seems inferable, but I might be too close to it so this is a good point.